packages/fast-usdc-deploy/src/fast-usdc-update.build.js

```diff
@@ -51,14 +54,25 @@ export const updateProposalBuilder = async (
 export default async (homeP, endowments) => {
   const { writeCoreEval } = await makeHelpers(homeP, endowments);
   const {
-    values: { feedPolicy },
+    values: { feedPolicy: feedPolicyJSON, network },
   } = parseArgs({ args: endowments.scriptArgs, options });
 
-  const parseFeedPolicy = () => {
-    if (typeof feedPolicy !== 'string') throw Error(feedPolicyUsage);
-    return JSON.parse(feedPolicy);
-  };
-  const config = harden({ feedPolicy: parseFeedPolicy() });
+  if (typeof feedPolicyJSON !== 'string') throw Error(feedPolicyUsage);
+  const feedPolicy = JSON.parse(feedPolicyJSON);
+  if (network !== undefined) {
+    if (typeof network !== 'string') {
+      throw Error('--network requires a string argument');
+    }
+    if (!Object.hasOwn(ChainPolicies, network)) {
+      const q = JSON.stringify;
+      throw Error(
+        `network name ${q(network)} not in ${q(Reflect.ownKeys(ChainPolicies))}`,
+      );
+    }
+    if (feedPolicy.chainPolicies) throw Error('cannot merge chainPolicies');
+    feedPolicy.chainPolicies = ChainPolicies[network];
```

Comment:
mutation is a little surprising and makes me wonder if I should re-check the whole scope of `feedPolicy` given this new info. That scope is only a few lines so it's no big deal, but I'd expect to see:

```suggestion
    feedPolicy = { ...feedPolicy, chainPolicies: ChainPolicies[network] };
```